packit_service/worker/helpers/testing_farm.py

```diff
@@ -219,6 +219,12 @@ def _payload(
             "PACKIT_SOURCE_URL": self.source_project_url,
             "PACKIT_TARGET_URL": self.target_project_url,
             "PACKIT_PR_ID": self.pr_id,
+            "PACKIT_COPR_PROJECT": f"{build.owner}/{build.project_name}"
+            if build
+            else None,
+            "PACKIT_COPR_RPMS": " ".join(artifact["packages"])
+            if artifact and artifact.get("packages")
+            else None,
```

Comment:
What about just `PACKIT_RPMS`? Or, do we want tight that to Copr?

Reply:
I guess this is about the future: if we fill the variable from other build systems (especially koji) - should we create a new variable or only use a single one?

I personally lean towards `PACKIT_RPMS` but it's true that `PACKIT_COPR_RPMS` is very explicit about the content.

Could be worth brainstorming on architecture. We can also set both :D